### ISSUE TITLE
Feature Request Windows: Allow Proxyman to run in a Split Tunnel while running Private Internet Access VPN

### ISSUE THREAD

**Issue by jlorber** [ProxymanApp/Proxyman#1509]
## Description

Please allow Proxyman for Windows to run properly in the Split Tunnel inside Private Internet Access VPN. 
## Why this feature/change is important?

This would allow both Proxyman and PIA VPN to run concurrently. I've tried this with Charles Proxy and it works correctly; however, Proxyman does not. I've included a screenshot and a link to PIA VPN for reference. Thank you!!

![Screenshot 2023-01-21 200938](https://user-images.githubusercontent.com/57275634/213898859-af8ba05e-ffab-47d8-968a-cce0a1dd4cdb.png)

Please click on the following link for more info:
https://helpdesk.privateinternetaccess.com/kb/articles/split-tunnel-app-examples

**Comment by NghiaTranUIT** [comment]
Not sure how to tackle it, if Charles Proxy works, Proxyman should work too because they both override the System Proxy. 

@jlorber when you mentioned that "Proxyman doesn't", you means Proxyman could not capture the traffic? 🤔

**Comment by jlorber** [comment]
I agree @NghiaTranUIT  - it should work the same...but when I turn on the PIA VPN then run Charles, everything is captured properly. When I turn on the PIA VPN and run Proxyman, only the Request data is captured-- mostly "Active" with a yellow status and no Response data. I see a bunch of 999 Codes with the Status of "Internal Error"  It's as if Proxyman is blocking the traffic flow for some reason... when the VPN is off, that doesn't seem to be an issue EXCEPT for one URL -- it has to do with a PINGFED login (only when SSL is enabled) and it won't pass the traffic either (Charles doesn't experience this issue either - it works properly and I can log in and see the data). If I Disable SSL proxying for that URL in Proxyman, then passes the traffic and I can log in... of course, that defeats the purpose of being able to see that data, because without SSL enabled, I can't. Again- this is a separate issue.

**Comment by NghiaTranUIT** [comment]
@jlorber can you share with me the URL of the `PINGFED login` or the app/website that I can test? You can send me via nghia@proxyman.io

I'm interested to fix this issue first.
